### PR TITLE
Add support for ThreatMetrix mobile native device profiling state

### PR DESCRIPTION
**Pull request by anandgopinath-ping**
* Add template for DEVICE_PROFILE_SESSION_ID_REQUIRED state

### CODE REVIEW

**Comment by khosravian** [comment]
Leaving this to @mketchum

File: src/partials/device_profile_session_id_required.hbs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo" />
+  </div>
+  <h1 class="heading">Session ID Cookie Not Found</h1>
+  <form class="form form--margin-small" id="AuthnWidgetForm">
+    <div class="text-block">
+      <div class="icon-feedback icon-feedback--error">
+        <span class="icon-feedback__icon pingicon-ghost-error icon-feedback--error"></span>
+      </div>
+      <div class="text-block--overflow-wrap">
+        A ThreatMetrix Session ID cookie was not found. Please configure the ThreatMetrix adapter according to
```
Comment (mketchum) [inline src/partials/device_profile_session_id_required.hbs:12]: This state isn't specific to ThreatMetrix anymore right?

Comment (anandgopinath-ping) [reply src/partials/device_profile_session_id_required.hbs:12]: Yes indeed. I will update.

**Comment by mketchum** [comment]
I'll let @cindytu-ping look over the wording as well to make sure it looks good

**Pull request merged**